specs/_features/eip7594/das-core.md

```diff
@@ -106,18 +106,20 @@ def get_custody_columns(node_id: NodeID, custody_subnet_count: uint64) -> Sequen
     assert custody_subnet_count <= DATA_COLUMN_SIDECAR_SUBNET_COUNT
 
     subnet_ids: List[uint64] = []
-    i = 0
+    i = uint256(node_id)
```

Comment:
Ah sorry @ralexstokes I think I have the revert the previous commit. There is another `i` usage in the `return sorted(...)`... so it's less good to assign an `i` here.